website/_redirects

```diff
@@ -0,0 +1,102 @@
+/ https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/
+/guide https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/
+/videos https://www.algolia.com/doc/guides/building-search-ui/getting-started/react/
+/Getting_started.html https://www.algolia.com/doc/guides/building-search-ui/getting-started/react/
+/Getting_started_React_native.html https://www.algolia.com/doc/guides/building-search-ui/going-further/native/react/
+
+# Widgets
+/widgets https://www.algolia.com/doc/api-reference/widgets/react/
+/widgets/<Index>.html https://www.algolia.com/doc/api-reference/widgets/index-widget/react
```

Comment:
The paths containing `<` characters don't seem to be redirected: https://deploy-preview-2021--react-instantsearch.netlify.com/widgets/%3CIndex%3E.html

Reply:
Yep indeed, that's annoying.